blocks/event-banner/event-banner.css

```diff
@@ -121,73 +118,121 @@
   pointer-events: none;
 }
 
+/* Force button spacing on mobile - override global 24px spacing */
+@media screen and (width <= 768px) {
+  .event-banner .event-banner-content p {
+    margin-bottom: 0 !important;
+  }
+  
+  /* Target button paragraph with strong specificity */
+  .event-banner .event-banner-content p:has(a),
```

Comment:
instead of relying on the order/number, you could also just target the button container class directly like this:
`.event-banner .event-banner-content .button-container to target the button`